_data/projects/coala-analyzer.yml

```diff
@@ -0,0 +1,16 @@
+name: coala-analyzer
```

Comment:
can we use the `coala` name? We only have coala-analyzer on github as coala was unavailable